CMakeLists.txt

```diff
@@ -379,14 +379,13 @@ if(OCOS_ENABLE_BLINGFIRE)
 endif()
 
 if(OCOS_ENABLE_AZURE)
-
   if ($ENV{TEST_AZURE_INVOKERS_AS_CPU_OP} MATCHES "ON")
-    message(STATUS "Azure inovkers will be testable as cpu ops")
+    message(STATUS "Azure invokers will be testable as cpu ops")
     add_compile_definitions(TEST_AZURE_INVOKERS_AS_CPU_OP)
   endif()
 
   # Azure endpoint invokers
-  include(triton)
+  include(triton)  # TODO: Allow curl and triton support to be added independently
```

Comment:
This could rename to azure.cmake to be more general.

Reply:
Will do at the end so the diffs are easier to see.